lib/rust/ensogl/core/src/gui/cursor.rs

```diff
@@ -303,8 +303,8 @@ impl Cursor {
             });
 
             eval frp.set_style([host_attached_weight,size,offset,model] (new_style) {
-                host_attached_weight.stop_and_rewind();
-                if new_style.host.is_some() { host_attached_weight.start() }
+                host_attached_weight.stop_and_rewind(0.0);
+                if new_style.host.is_some() { host_attached_weight.target(1.0) }
```

Comment:
just double checking - are you 100% sure it works the exactly same way as the previous version?

Reply:
I made this change a while ago, but I remember this being functionally equivalent. We should carefully test that this has no corner cases, but otherwise, yes this should be fine.